core/modules/filter/filter.admin.inc

```diff
@@ -564,7 +564,7 @@ function filter_editor_image_upload_settings_form($format) {
     '#type' => 'textfield',
     '#default_value' => $settings['directory'],
     '#title' => t('Upload directory'),
-    '#description' => t("A directory relative to the files directory where uploaded images will be stored."),
+    '#description' => t("A directory relative to the files directory where uploaded images will be stored.<br />This field supports tokens.") . ' ' . theme('token_tree_link'),
```

Comment:
Let's wrap `This field supports tokens.` in it's own `t()` call. That string should already be translated and we won't have to modify the current string.